### PR TITLE
WIP [fix bug 1233779] Serve specific builds to WinXP users.

### PR DESCRIPTION
**Pull request by jpetto**
Still working through some CSS/display issues, but hoping for a review/sanity check on direction.

@alexgibson, @jgmize, @pmclanahan - Care to take a look?

### CODE REVIEW

File: media/js/base/site.js [hunk 1]
```diff
@@ -152,6 +153,11 @@
             if (version && parseFloat(version) >= 6.1) {
                 h.className += ' win7up';
             }
+
+            // Check for Windows XP users to alter download button behavior
+            if (/Windows NT 5\.1/i.test(navigator.userAgent)) {
```
Comment (pmclanahan) [inline media/js/base/site.js]: The [logic from the bouncer efforts](https://github.com/mozilla-services/go-bouncer/pull/16/files#diff-31a4a7824225629df071a1cba8b785d5R26) also looks for "XP". Not sure if this is a real thing, but might be worth checking for as well.

**Comment by pmclanahan** [comment]
The direction seems good. Not sure we'll need it yet or not. It's looking like bouncer will roll out their changes soon, but it's great that we're ready to go if that effort fails.

**Comment by jpetto** [comment]
Looks like the bouncer fix did the trick. Closing this PR, but will keep the branch around just in case.

**Comment by pmclanahan** [comment]
Sounds good. Thanks for the great work @jpetto, we may yet need this or something like it.

**Pull request closed without merging**